content/_index.md

```diff
@@ -37,7 +37,7 @@ In an estimated minimum of a day (~5.5 hours) you can achieve the following:
 - **Track -** Set-up and deploy tracking needed for your website performance data
 - **Enrich -** Add an enrichment to your data
 - **Model -** Configure and run the [core web vitals](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-web-data-model//core-web-vitals-module/) module of the `snowplow-web` data model
-- **Visualize -** Use the Tableau twbx template (csv based!) to gain insight into your own website's health
+- **Visualize -** Use the Tableau twbx template (csv based!) or the Preset dashboard template to gain insight into your own website's health
 - **Next steps -** Gain more in-depth knowledge on web performance with additional metrics
```

Comment:
Just and idea but here we only give a sneak peek into the tableau dashboard here, can be add the other image as well, perhaps in a split tab view?